MetaMask/metamask-extension
# Added ONE logo

Fixes: #
Adding Harmony ONE will show logo

Explanation:  

Manual testing steps:  
  - Add Harmoney Network with ONE currency
  - Check currency logo is showing
